### PR TITLE
Allow override cluster size plus observers

### PR DESCRIPTION
**Pull request by rgs1**

### CODE REVIEW

**Comment by rgs1** [comment]
the failing tests look unrelated

File: kazoo/testing/common.py [hunk 1]
```diff
@@ -263,7 +264,12 @@ def __init__(self, install_path=None, classpath=None,
         peers = []
 
         for i in range(size):
-            info = ServerInfo(i + 1, port, port + 1, port + 2, port + 3)
+            server_id = i + 1
+            if observer_start_id != -1 and server_id >= observer_start_id:
+                peer_type = 'observer'
```
Comment (Yasumoto) [inline kazoo/testing/common.py]: Make these constants?

**Comment by Yasumoto** [comment]
Overall seems good to me

**Comment by rgs1** [comment]
ping @bbangert @harlowja :-)

**Comment by harlowja** [comment]
:+1: 

cool, looks ok to me

for those who don't know,

http://zookeeper.apache.org/doc/trunk/zookeeperObservers.html

**Comment by rgs1** [comment]
ping @harlowja  :-)

**Comment by harlowja** [comment]
Pong.

**Pull request merged**